openmetadata-ui/src/main/resources/ui/src/components/Settings/Users/Users.component.tsx

```diff
@@ -267,12 +263,12 @@ const Users = ({
 
   const descriptionRenderComponent = useMemo(
     () =>
-      hasEditPermission ? (
+      isLoggedInUser ? (
         <DescriptionV1
           description={userData.description ?? ''}
           entityName={getEntityName(userData as unknown as EntityReference)}
           entityType={EntityType.USER}
-          hasEditAccess={hasEditPermission}
+          hasEditAccess={isLoggedInUser}
```

Comment:
Isn't admin should also have this access? or we want to limit it to users only

Reply:
We want only the profile owners to be able to edit the display name and description. Admins can edit every thing else